src/imagej/dims.py

```diff
@@ -467,3 +475,56 @@ def _to_ijdim(key: str) -> str:
         return ijdims[key]
     else:
         return key
+
+
+def _cal_axis_to_str(key) -> str:
+    """
+    Convert a CalibratedAxis class (e.g. net.imagej.axis.DefaultLinearAxis) to
+    a string.
+    """
+    cal_axis_to_str = {
+        jc.ChapmanRichardsAxis: "ChapmanRichardsAxis",
+        jc.DefaultLinearAxis: "DefaultLinearAxis",
+        jc.EnumeratedAxis: "EnumeratedAxis",
+        jc.ExponentialAxis: "ExponentialAxis",
+        jc.ExponentialRecoveryAxis: "ExponentialRecoveryAxis",
+        jc.GammaVariateAxis: "GammaVariateAxis",
+        jc.GaussianAxis: "GaussianAxis",
+        jc.IdentityAxis: "IdentityAxis",
+        jc.InverseRodbardAxis: "InverseRodbardAxis",
+        jc.LogLinearAxis: "LogLinearAxis",
+        jc.PolynomialAxis: "PolynomialAxis",
+        jc.PowerAxis: "PowerAxis",
+        jc.RodbardAxis: "RodbardAxis",
+    }
+
+    if key.__class__ in cal_axis_to_str:
+        return cal_axis_to_str[key.__class__]
+    else:
+        return "unknown"
+
+
+def _str_to_cal_axis(key: str):
+    """
+    Convert a string (e.g. "DefaultLinearAxis") to a CalibratedAxis class.
+    """
+    str_to_cal_axis = {
+        "ChapmanRichardsAxis": jc.ChapmanRichardsAxis,
+        "DefaultLinearAxis": jc.DefaultLinearAxis,
+        "EnumeratedAxis": jc.EnumeratedAxis,
+        "ExponentialAxis": jc.ExponentialAxis,
+        "ExponentialRecoveryAxis": jc.ExponentialRecoveryAxis,
+        "GammaVariateAxis": jc.GammaVariateAxis,
+        "GaussianAxis": jc.GaussianAxis,
+        "IdentityAxis": jc.IdentityAxis,
+        "InverseRodbardAxis": jc.InverseRodbardAxis,
+        "LogLinearAxis": jc.LogLinearAxis,
+        "PolynomialAxis": jc.PolynomialAxis,
+        "PowerAxis": jc.PowerAxis,
+        "RodbardAxis": jc.RodbardAxis,
+    }
```

Comment:
Can we avoid duplicating this map? It should be unique keys and unique values, so you could write something like
```python3
cal_axis_to_str=<what you already have>
str_to_cal_axis = {v:k for k, v in f.items()}
```

Reply:
I am not longer using a dict to store the `CalibratedAxis` with its string. I ran into issues where if a you try to import the `metadata` module without starting Jpype first (_i.e._ initializing PyImageJ) you hit a jvm not started exception. This makes sense but is annoying to work around just to keep the dict. Instead I'm storing I decided to store the endpoint and then call `imagej._java.jc` helper module to handle the imports (see 56ca2724a7f1a88fa30648d76c43427827a8aa49).